PWGHF/vertexingHF/AliAnalysisTaskSELc2pKs0fromKFP.h

```diff
@@ -147,6 +148,7 @@ class AliAnalysisTaskSELc2pKs0fromKFP : public AliAnalysisTaskSE
         Bool_t                  fIsMC; ///< Flag of MC analysis
         Bool_t                  fUseWeights; ///< Flag to use pT weight functions
         Bool_t                  fKeepOnlyMCSignal; ///< flag to keep only signal candidates
+        Bool_t                  fKeepAllVariables; ///<flag to keep all possible variables that were removed to reduce the tree size
```

Comment:
Every time you add a data member you should update the `ClassDef` (in this case from `7` to `8`)

Reply:
Thanks a lot for spotting this. I did not see this.